Containerfile

```diff
@@ -16,6 +16,13 @@ FROM $PYTHON_BASE_IMAGE
 COPY --from=python_builder /output/ /output
 RUN /output/install-from-bindep && rm -rf /output
 
+RUN dnf update -y \
+  && dnf install -y python38-wheel git \
+  && dnf clean all \
+  && rm -rf /var/cache/{dnf,yum} \
+  && rm -rf /var/lib/dnf/history.* \
+  && rm -rf /var/log/*
+
```

Comment:
actually, this isn't needed. Because we are using python-builder to build this image. We can add them into the bindep.txt file, and things will just work properly.